### ISSUE TITLE
guiAddCards vs. addNote

### ISSUE THREAD

**Issue by mzhou08** [FooSoft/anki-connect#275]
As a new user of Anki, I was trying to use ankiConnect to automatically add cards to my decks without opening any dialogs. I found the guiAddCards function, and tried to modify it to not show dialogs. However, I found the addNote function, which accomplishes exactly what I wanted.

To prevent similar confusion for future users, I've slightly modified the guiAddCards command to include an autoAdd option, which directly calls addNote. I've updated the README accordingly, adding a few lines to guiAddCards's description. I've also enabled silent closing for the closeAfterAdding.

Thanks for developing such a comprehensive Anki API!

**Comment by mzhou08** [comment]
Please review my pull request, linked [here.](https://github.com/FooSoft/anki-connect/pull/276)

**Comment by uzqw** [comment]
yes, I have searched for the function of adding cards without GUI for so long, until I find your PR. "Add notes" really confused me, I didn't know it was what I need.
